change/@ni-nimble-components-8cad4d26-536b-4f4b-890f-233d13b36bd0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Update box shadow tokens, including removing the 'popupBoxShadowColor' token",
```

Comment:
Could you add guidance about what clients should use instead of `popupBoxShadowColor`? e.g. pointing them to the new "elevation*BoxShadow" tokens.

Reply:
Sure. I didn't add guidance because we've been using `popupBoxShadowColor` for pretty much everything, so there isn't a specific new token that they should use. But, I can generically point to the new tokens.